src/components/CTypeDetails/__snapshots__/CTypeDetails.test.ts.snap

```diff
@@ -121,7 +121,7 @@ exports[`CTypeDetails > should handle kitchen sink CType 1`] = `
   </fieldset>
   <fieldset>
     <legend>Registration Block</legend>
-    <a href="https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fplaceholder#/explorer/query/456">456</a>
+    <a href="https://polkadot.js.org/apps/?rpc=ws://localhost:32769#/explorer/query/456">456</a>
```

Comment:
See, this is an invalid URL

Reply:
This is a valid URL. I just tried it out with a local chain.